app/models/full_refresh_upload_task.rb

```diff
@@ -0,0 +1,52 @@
+class FullRefreshUploadTask < MqttClientTask
+  RESPONSE_FORMAT = "tar".freeze
+
+  def content(_options)
+    payload = CatalogInventory::Payload.new(RESPONSE_FORMAT, upload_url, full_refresh_jobs)
+    self[:input] = payload.as_json
+
+    payload.as_json
+  end
+
+  def upload_url
+    ENV.fetch("UPLOAD_URL") || raise("UPLOAD_URL must be specified")
+  end
+
+  def full_refresh_jobs
+    jobs = []
+
+    job_template = CatalogInventory::Job.new
```

Comment:
try to be consistent. job_template or job_templates? Other types are in plural in the context.